src/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs

```diff
@@ -271,6 +280,42 @@ public async Task PutToCountryLocalNames_ShouldUpdateCountryLocalName()
         }
         #endregion
 
+        #region PUT ZeroOrOne Owned Entity /api/{EntityPluralName}/{key} => api/countries/1
+        [Fact]
+        public async Task Put_WithCountryBarCode_ShouldUpdateCountryBarCode()
+         {
+            // Arrange
+            var expectedBarCode = _fixture.Create<string>();
+            var dto = new CountryCreateDto
+            {
+                Name = _fixture.Create<string>(),
+                CountryBarCode = new CountryBarCodeCreateDto() { BarCodeName = _fixture.Create<string>() }
+            };
+            // Act
+            var postCountryResponse = await _oDataFixture.PostAsync<CountryCreateDto, CountryDto>(EntityUrl, dto);
+            var headers = _oDataFixture.CreateEtagHeader(postCountryResponse?.Etag);
+            var putCountryResponse = await _oDataFixture.PutAsync<CountryUpdateDto, CountryDto>(
+                $"{EntityUrl}/{postCountryResponse!.Id}",
+                new CountryUpdateDto
+                {
+                    Name = postCountryResponse!.Name,
+                    CountryBarCode = new CountryBarCodeUpdateDto() { BarCodeName = expectedBarCode }
+
+                }, headers);
+
+            var getCountryResponse = await _oDataFixture.GetODataSimpleResponseAsync<CountryDto>($"{EntityUrl}/{postCountryResponse!.Id}");
+
+            //Assert
+            putCountryResponse.Should().NotBeNull();
+            putCountryResponse!.Id.Should().Be(postCountryResponse!.Id);
+            putCountryResponse!.Name.Should().Be(postCountryResponse!.Name);
+
+            getCountryResponse!.Id.Should().Be(postCountryResponse!.Id);
+            getCountryResponse!.Name.Should().Be(postCountryResponse!.Name);
+            //getCountryResponse!.CountryBarCode!.BarCodeName.Should().Be(expectedBarCode);
```

Comment:
@rochar need to fix PUT and PATCH for single owned entities, it doesn't work now - #772

Reply:
ok